### DISCUSSION TITLE
How to bundle app with internal packages not symlinked

### DISCUSSION THREAD

**Discussion by grug** [vercel/turborepo#3817] [Help]
Hey there, if I have an app that uses internal packages (let's say A and B), they show up in node modules as symlinks. This is fine for local development, but when I want to do a production build of this application (just a node/express application), I need to build my code (which outputs to a dist directory) and doesn't contain my node modules in there, so I add an extra step to copy my node modules into dist (so that I can zip up the directory and send it to Azure App Services) but the internal packages don't also copy since they are symlinks in my local project.

I'd like to get to a state where I can zip up my dist directory, unzip it elsewhere and run `node server.js` and it has all the node modules required to run it at its deployed destination.

I'm hoping I've explained that well enough so that I can get some guidance on what to do about this.

**Comment by gtjamesa** [comment]
If you're using pnpm, you can use [`pnpm deploy`](https://pnpm.io/cli/deploy)

**Comment by mirolyubovN** [comment]
@grug did you manage to find the solution? I am facing the same issue when trying to deploy the app to Azure

**Reply by meyer0x** [reply]
same here
